src/js/services/srv-widget.js

```diff
@@ -233,6 +233,10 @@ app.service('widgetSrv', function(selectedWidgetSrv, dateTimeServ, $q, $rootScop
 		}
 	}
 
+	const canBePublishedByCurrentUser = widget_id => {
```

Comment:
This doesn't need to change here, but this line *can* be a single line using the single line return syntax

```suggestion
	const canBePublishedByCurrentUser = widget_id => selectedWidgetSrv.getPublishPermission(widget_id)
```